### DISCUSSION TITLE
Week 1: Introduction to DataScience

### DISCUSSION THREAD

**Discussion by tejuafonja** [AISaturdaysLagos/cohort_structure#4] [General]
**Hi everyone, you can use this thread to discuss week 1 content. Below are some suggested topics**

- Cool Data Science applications!!!😎
  - Link to the application 
  - Why do you think it's cool

- Today I learned ✨
-  ...

**Comment by shadynen** [comment]
Thank will discuss better subsequently.

**Comment by thechinonso** [comment]
Happy to have kicked off on a good note! It's not always easy to clearly tell the difference between data science and machine learning. The class was very informative in breaking down all complexities into fine detail. Data science takes a more holistic approach (computation + statistics) to give insights into a problem (or to answer a question). Machine learning mostly involves writing algorithms that use data to make near-accurate predictions. 

Looking forward to the next session!
